aten/src/ATen/test/CMakeLists.txt

```diff
@@ -44,7 +44,6 @@ list(APPEND ATen_CPU_TEST_SRCS
   ${CMAKE_CURRENT_SOURCE_DIR}/type_ptr_test.cpp
   ${CMAKE_CURRENT_SOURCE_DIR}/type_test.cpp
   ${CMAKE_CURRENT_SOURCE_DIR}/undefined_tensor_test.cpp
-  ${CMAKE_CURRENT_SOURCE_DIR}/variant_test.cpp
```

Comment:
Um, is this intentional? We should wait until we actually delete the variant_test.cpp?

Reply:
I forgot to revert it. I couldn't delete variant.h because the file contains 3K lines and PR sanitizer check failed.